platforms/org.eclipse.mita.platform.xdk110/src/org/eclipse/mita/platform/xdk110/connectivity/WlanGenerator.xtend

```diff
@@ -200,5 +209,15 @@ class WlanGenerator extends AbstractSystemResourceGenerator {
 		}
 		return result
 	}
-	
-}
\ No newline at end of file
+	private def CodeFragment buildServiceCallback(SystemResourceSetup component, Iterable<EventHandlerDeclaration> declarations) {
+						val baseName = component.baseName
```

Comment:
This differs from line 50. Can you please explain (via `//`-comments) why?

Reply:
Line 50 was not needed hence removed